src/main/java/frc/shooter/ShooterSensors.java

```diff
@@ -0,0 +1,43 @@
+package frc.shooter;
+
+import frc.sensors.Limelight;
+import frc.shooter.MathHelpers;
+import frc.drives.DrivesSensors;
+
+public class ShooterSensors implements ShooterSensorsInterfeace 
+{
+	Limelight lightSensor; //Used for accessing member methods of the limelight.
+	DrivesSensors driveSensor; //Used for accessing member methods of the drive sensors.
+	double cameraToRobotAngle;
+	public ShooterSensors(double cameraToRobotAngle, Limelight light, DrivesSensors drive)
+	{
+		this.lightSensor = light;

```

Comment:
Create an instance of the limelight sensor in here. No need to pass it in. Shooter is the only subsystem which uses that sensor, otherwise it would make sense to pass it in.